lib/build/source-inclusion.js

```diff
@@ -27,8 +28,16 @@ exports.SourceInclusion = class {
     this.items.push(item);
   }
 
+  isExcluded(item){
+	  let found = this.excludes.findIndex(x => {			
+			let exclusion = new this.bundle.Minimatch(x);
```

Comment:
I'm mainly wondering if this could be more efficient by creating the Minimatch objects once, upfront, rather than on every call to isExcluded. @AStoker Any thoughts on that?